.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: r
+cache: packages
+before_install:
+  - cd dataCompareR
```

Comment:
Without this, Travis doesn't recognise that its an R package, and can't find the unit tests to run.